packages/dashboard/src/customization/widgets/utils/assetModelQueryToAssetQuery.ts

```diff
@@ -0,0 +1,46 @@
+import { AssetModelQuery, SiteWiseAssetQuery } from '@iot-app-kit/source-iotsitewise';
+
+import { unionBy, uniq } from 'lodash';
```

Comment:
Does this mean we are pulling in the entire Lodash library?

Reply:
👍 updated import to supported tree shaking syntax.